core/src/test/scala/integration/kafka/server/FetchFromFollowerIntegrationTest.scala

```diff
@@ -225,7 +225,7 @@ class FetchFromFollowerIntegrationTest extends BaseFetchRequestTest {
 
       val recordFutures = consumers.zipWithIndex.map { case (consumer, i) =>
         executor.submit(() => {
-          TestUtils.pollUntilAtLeastNumRecords(consumer, assignments(i).size, waitTimeMs = 30000)
+          TestUtils.pollUntilAtLeastNumRecords(consumer, assignments(i).size, waitTimeMs = 30000, commitOffset = true)
```

Comment:
could we commit after `TestUtils.pollUntilAtLeastNumRecords` returns?

Reply:
Updated